src/client/modules/content/ContentWrapper.jsx

```diff
@@ -1,16 +1,18 @@
 /* global window */
 import React from 'react';
-import {Router, Route, hashHistory, browserHistory} from 'react-router';
-import ProjectsPage from './pages/ProjectsPage.jsx';
 
 export default class ContentWrapper extends React.Component {
 
     constructor(props) {
         super(props);
+        this.restClient = this.props.restClient;
```

Comment:
if you will only use the restClient, then I suggest not creating an extra reference to it (using always from props should be fine)